### ISSUE TITLE
Config for the maximum height of the snow

### ISSUE THREAD

**Issue by SprutSDM** [kangarko/Winter#97]
Hi! Thank you for the plugin—I've been using it for 4 years in a row, and it has been fantastic.

I have a small feature request: could you please add a configuration option to allow increasing the maximum height of the snow? I would like to create abnormally heavy snow in my server. Currently, the maximum height seems to be capped at 4. It would be perfect if this could be raised to 16 or even more, enabling the generation of complete blocks of snow.

**Comment by kangarko** [comment]
Hey and thanks for the kind words - four years that's a lot! I actually just finished a big ChatControl update and taking vacation soon, but if I find time tomorrow I will get to this :)

**Comment by kangarko** [comment]
Hey! Just a quick update I looked into it and it would require more of my time than I unfortunately have these holidays, to make sure the placement will stop on a given limit. If you don't mind then I can have a look after Jan 3, hope you understand!

**Comment by SprutSDM** [comment]
Thats fine, Merry Christmas! 🎄

**Comment by kangarko** [comment]
Should be finally done, thanks for the wait!

**Comment by SprutSDM** [comment]
Hi! Thanks so much for adding this feature, I really appreciate it! 🎉

I just tried out the new version, and it mostly works great. But I noticed it only seems to work if I set Required_Neighbors_To_Grow to 0. Am I missing something? If the value is higher, it seems like the middle block doesn’t grow when the neighbors have more snow, which stops the growth.

Maybe it could be possible to let the central snow grow even if the neighbors are at the same or a higher level (right now, it grows only if they’re the same level, but not if they’re higher), as long as the Max_Snow_Height hasn’t been reached? That way, we could still get full blocks of snow while keeping the current 'wavy' pattern on top, avoiding a completely flat layer of snow.

What do you think about it? 😃

**Comment by kangarko** [comment]
Yep, it needs Required Neighbors to be 0. I will put this into the docs. Unfortunately this is a hobby plugin, I wont be able to have more time required for a thorough rewrite to accomplish what you're looking for entirely.

**Comment by SprutSDM** [comment]
Got it! Is there any chance I can contribute to the plugin? I have experience with Java/Kotlin and have written a few plugins before.

**Comment by kangarko** [comment]
Absolutely! That would be amazing! Please email matej@mineacademy.org with your preferred email address and I will add you to the repo we have at BitBucket.

Actually, on that note would you be open to contributing to ChatControl (paid)? Lets discuss via email.
